src/heartratetask/HeartRateTask.cpp

```diff
@@ -5,8 +5,25 @@
 
 using namespace Pinetime::Applications;
 
-HeartRateTask::HeartRateTask(Drivers::Hrs3300& heartRateSensor, Controllers::HeartRateController& controller)
-  : heartRateSensor {heartRateSensor}, controller {controller} {
+TickType_t CurrentTaskDelay(HeartRateTask::States state, TickType_t ppgDeltaTms) {
+  switch (state) {
+    case HeartRateTask::States::ScreenOnAndMeasuring:
+    case HeartRateTask::States::ScreenOffAndMeasuring:
+      return ppgDeltaTms;
+    case HeartRateTask::States::ScreenOnAndStopped:
+      return pdMS_TO_TICKS(100);
+    case HeartRateTask::States::ScreenOffAndWaiting:
+      return pdMS_TO_TICKS(10000);
```

Comment:
Does a 10 second delay cause considerable jitter for the 30 second option?

Reply:
Changed to `1000ms`